web/ui/src/features/component/style.ts

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
```

Comment:
Is this all for `react-syntax-highlighter` maybe a comment to mention it all works together or export as syntaxHighligher

Reply:
Good point. Yes, this is an entire object to pass in the `react-syntax-highlighter`, and it is pulled from the library's style object with some custom changes